app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UsersController < ApplicationController
+  before_action :require_user
+
+  def edit
+  end
+
+  def destroy
+    @user.destroy
+    flash[:notice] = "Account successfully deleted!"
+    redirect_to root_path
+  end
+
+  def update
+    if @user.email != params[:user][:email]
```

Comment:
Priekš kam tev vajag parbaudit kā emails mainijas?

Reply:
savadak vins aizmet uz root un flasho notice ka esot aizsutits email. Nevaig jau sutit email ja izmainas nav.

Reply:
Es pabakstijos kodā un dokos.

Tak vot, šo nosacijumu vari mierigi ņemt arā - tāpēc ka nakotnē tev būs vel lauki un visus līkt šajā condition nebūtu forši.
Un tev nevajag iekš Notice rakstit kā Epasts nosutits...

Tev vajadzejtu izdārīt visu pēc klasiska piemēra:
```ruby
if @user.update(user_params)
  flasj[:notice] = "Profile updated"
  ...
else
 ...
end
```

Es saprotu kā problēma ir šeit, kā Devise sūta confirmation vēstuli katru reizi kad tiek mainīts epasts (notestēju to kad pievienoju vairāk lauku formai).
To tev vajag vnk atslēgt no default configa, tāpēc kā pagaidam ši fiča mums nav vajadziga un ļoti mulsina.
https://stackoverflow.com/questions/9778452/how-to-skip-the-need-to-confirm-an-email-address-update-with-devise/9780008

Reply:
Done!